ramls/schemas/fileDefinition.json

```diff
@@ -36,6 +36,16 @@
       "description": "Size of the file in Kbyte",
       "type": "integer"
     },
+    "format": {
+      "description": "Format of exported file",
+      "type": "string",
+      "enum": [
+        "csv",
+        "cql",
+        "mrc"
```

Comment:
I think upload file format can not be mrc

Reply:
commented above